cli/main.c

```diff
@@ -1458,6 +1458,17 @@ static void print_fw_part_line(const char *tag,
 	       inf->valid ? "" : "  (Invalid)");
 }
 
+static void print_seeprom_part_line(const char *tag,
+				    struct switchtec_fw_image_info *inf)
+{
+	if (!inf)
+		return;
+
+	printf("  %-4s\tVersion: %-8s\tCRC: %08lx\t%s\n",
+	       tag, inf->version, inf->image_crc,
+	       inf->valid ? "" : "  (Invalid)");
```

Comment:
Why not just use print_fw_part_line()? We know read_only, running and redundant should be false don't we?

Reply:
Good idea. Updated.